### PR TITLE
[iOS] Add sendable conformance to event map feature

### PR DESCRIPTION
**Pull request by woxtu**
## Issue
- in progress #431

## Overview (Required)
- This PR adds sendable conformance to the event map feature and resolves the following warning:

```
Passing argument of non-sendable type 'EventMapReducer.Action' into main actor-isolated context may introduce data races
```

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

File: app-ios/Sources/Model/Entity/EventMapEvent.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public struct EventMapEvent: Equatable, Sendable, Hashable {
+    public let name: MultiLangText
+    public let roomName: MultiLangText
+    public let roomIcon: RoomIcon
+    public let description_: MultiLangText
```
Comment (ry-itto) [inline app-ios/Sources/Model/Entity/EventMapEvent.swift:7]: I think this `_` isn't needed for iOS side code 👀 

```suggestion
    public let description: MultiLangText
```

**Review by ry-itto — COMMENTED** [review]
Thanks!
Please check a comment 🙏🏼

File: app-ios/Sources/EventMapFeature/EventItem.swift [hunk 2]
```diff
@@ -2,9 +2,10 @@ import SwiftUI
 import CommonComponents
 import Theme
 import shared
```
Comment (MrSmart00) [inline app-ios/Sources/EventMapFeature/EventItem.swift:4]: Can you remove this import with your modified?
I guess Model package wrapped shared package, it makes shared imports to be no needed.

File: app-ios/Sources/EventMapFeature/EventMapReducer.swift [hunk 3]
```diff
@@ -2,7 +2,7 @@ import ComposableArchitecture
 import KMPClient
 import Model
 import Foundation
-@preconcurrency import shared
+import shared
```
Comment (MrSmart00) [inline app-ios/Sources/EventMapFeature/EventMapReducer.swift:5]: Can you remove this import with your modified?
I guess Model package wrapped shared package, it makes shared imports to be no needed.

**Review by MrSmart00 — COMMENTED** [review]
Thanks for your commits!
I left some comments.
Please check them🙏

File: app-ios/Sources/Model/Entity/EventMapEvent.swift (hunk 1 above)
Comment (woxtu) [reply app-ios/Sources/Model/Entity/EventMapEvent.swift]: I've renamed it.

File: app-ios/Sources/EventMapFeature/EventMapReducer.swift (hunk 3 above)
Comment (woxtu) [reply app-ios/Sources/EventMapFeature/EventMapReducer.swift]: Good point!

**Pull request merged**